hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -158,11 +158,12 @@ public function upgradeBundle() {
     CRM_Core_DAO::executeQuery("DROP TABLE IF EXISTS civicrm_hrhours_location");
     CRM_Core_DAO::executeQuery("
         CREATE TABLE IF NOT EXISTS `civicrm_hrhours_location` (
-        `id` int(10) unsigned NOT NULL,
+          `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
If we're adding this to an existing upgrader won't it be a problem for installations where this upgrader has already been run?